docs/manual/developer/06_contributing_with_content.md

```diff
@@ -418,15 +418,12 @@ that begin with underscores are not meant to be used in descriptions.
 You can also check documentation for all macros in the `Jinja Macros Reference`
 section accessible from the table of contents.
 
-To parametrize rules and remediations as well as Jinja macros, you can
-use product-specific variables defined in `product.yml` in product root
-directory. Moreover, you can define **implied properties** which are
-variables inferred from them. For example, you can define a condition
-that checks if the system uses `yum` or `dnf` as a package manager and
-based on that populate a variable containing correct path to the
-configuration file. The inferring logic is implemented in
-`_get_implied_properties` in `ssg/yaml.py`. Constants and mappings used
-in implied properties should be defined in `ssg/constants.py`.
+To parametrize rules and remediations as well as Jinja macros,
+use product-specific variables defined either
+in `product.yml` in product root directory,
+or in files in the `product_properties` project directory.
+Use that directory to create new product properties to avoid relating checks to product versions -
+instead realte product versions to product properties, and product properties to checks.
```

Comment:
It's hard to understand.

From the file `product_properties/20-grub.yml` it seems that there actually can be a relation between product and product properties.

Maybe a different wording or an example would be great to make this documentation better understandable.

Reply:
I have reformulated, check whether it is better.